service/controller/key/common.go

```diff
@@ -280,3 +280,7 @@ func imageIDs() map[string]string {
 func isChinaRegion(region string) bool {
 	return strings.HasPrefix(region, "cn-")
 }
+
+func HealthCheckTarget(port int) string {
```

Comment:
So far the order was first public and then private methods. Both sections sort alphabetically as well if no other stronger reason. Having this consistency helps humans find anything in arbitrary projects.